app/routes/payments/$paymentId.tsx

```diff
@@ -23,17 +728,366 @@ export const loader: LoaderFunction = async ({ params }) => {
+      timestamp={date}
+      format="DD/MM/YYYY"
+      color={theme.palette.neutral[600]}
+    />
+  </Box>
+);
+
+interface PaymentDetailsLoaderWrap {
+  details: PaymentDetail;
+}
+
+export default function PaymentDetails() {
+  const { details } = useLoaderData<PaymentDetailsLoaderWrap>();
   const { t } = useTranslation();
 
   return (
-    <Page id="payment" title={t('pages.payment.title')}>
-      <Typography>Id : {payment.id}</Typography>
+    <Page
+      id="payment"
+      title={titleHeader(t('pages.payment.title'), details.createdAt)}
+      onClick={() => null}
+      actionEvent="add_lorem_action"
+      actionLabel="View on Stripe dashboard"
+      actionId="add-lorem"
+    >
+      <Box
+        sx={{
+          display: 'flex',
+          flexDirection: 'column',
+          padding: '20px',
+          backgroundColor: 'white',
+        }}
+      >
+        <Box
+          sx={{
+            display: 'flex',
+            'justify-content': 'space-between',
+            width: '100%',
+            'padding-bottom': '20px',
+          }}
+        >
+          {boxWithCopyToClipboard(
+            t('pages.payment.id'),
+            details.id,
+            theme.palette.blue.light
+          )}
+          {boxWithCopyToClipboard(
+            t('pages.payment.reference'),
+            details.id,
+            theme.palette.violet.light
+          )}
+        </Box>
+        <DividerWithSpace />
+        <Box
+          sx={{
+            display: 'flex',
+            width: '100%',
+            justifyContent: 'space-around',
+          }}
+        >
+          <Box
+            sx={{
+              display: 'flex',
+              width: '45%',
+              flexDirection: 'column',
+              justifyContent: 'start',
+            }}
+          >
+            {dataItem(
+              t('pages.payment.type'),
+              chipsWithColor(
+                'Pay-in',
+                theme.palette.green.light,
+                theme.palette.green.darker
+              )
+            )}
+            {dataItem(
+              t('pages.payment.processor'),
+              <Box sx={{ m: '10px' }}>{details.provider}</Box>
+            )}
+            {dataItem(
+              t('pages.payment.status'),
+              chipsWithColor(
+                details.status,
+                theme.palette.green.light,
+                theme.palette.green.darker
+              )
+            )}
+          </Box>
+
+          <Box
+            sx={{
+              display: 'flex',
+              width: '45%',
+              flexDirection: 'column',
+            }}
+          >
+            {dataItem(
+              t('pages.payment.netValue'),
+              <>{details.initialAmount}</>
+            )}
+            {dataItem(
+              t('pages.payment.initialAmount'),
+              <>{details.initialAmount}</>
+            )}
+          </Box>
+        </Box>
+        <DividerWithSpace />
+
+        <Typography variant="headline">
```

Comment:
You could use SectionWrapper https://62dfdffc8257731cc208cedd-pxduxrovom.chromatic.com/?path=/docs/design-system-containers-wrapper--default-section-wrapper